cache-dependency-binary/index.js

```diff
@@ -35,6 +31,10 @@ async function handleRequest(request) {
     }})
   }
 
+  if (version[0] == 'v') {
```

Comment:
why not just leave this where it was, and include the `v` for M1 binaries too? or am I misinterpreting this

Reply:
yeah as-is, this would mean the m1 path wouldn't support requesting without the `v` for the version, while the normal path to get from github would support both `vX.X.X and `X.X.X`. It'd be nice to have the same behavior, I'm indifferent to how it happens

Reply:
good catch -- will update